src/blog/v8-release-80.md

```diff
@@ -0,0 +1,136 @@
+<!-- markdownlint-restore -->
```

Comment:
needs a line break after this comment

Reply:
that's actually intentionally not there to avoid whitespace after the image.

Reply:
meh, negative margins is how I solved the top margin.